client/src/pages/Profile.jsx

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { getAttendingSessions } from '../utils/apiWrapper';
+import 'semantic-ui-css/semantic.min.css';
+import '../css/Profile.scss';
+import SessionSummary from '../components/SessionSummary';
+import ClassForm from '../components/ClassForm.jsx';
 import ClassCard from '../components/ClassCard';
```

Comment:
Nit - Put lines 4-5 after lines 6-8. CSS imports should go after component imports